Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,38 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name1 = ("Tom Jefferson");
+            int age1 = 19;
```

Comment:
One more change is needed- pleas read from console. Refer to ``Console.ReadLine`` ``inter.Parse`` double.Parse``

Reply:
not entirely sure what that means?

Reply:
instead of 
```
            double weight1 = 50;
            double height1 = 156.5;
```
You need to input those numbers using console. because user won't be changing the code.
So it should be:
```
            double weight1 = double.Parse(Console.ReadLine());
            double height1 = double.Parse(Console.ReadLine());
```